addon/ng2/blueprints/ng2/files/__path__/tsconfig.json

```diff
@@ -1,24 +1,23 @@
 {
-  "compileOnSave": false,
   "compilerOptions": {
+    "baseUrl":"./src",
     "declaration": false,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "mapRoot": "/",
```

Comment:
What's going on here? We don't care about TypeScript maps?  Even so, we probably want to support running `tsc` and having source maps.